mmv1/products/logging/LinkedDataset.yaml

```diff
@@ -0,0 +1,123 @@
+# Copyright 2023 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Resource
+name: "LinkedDataset"
+base_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/links'
+create_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/links?linkId={{link_id}}'
+delete_url: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/links/{{link_id}}'
+self_link: '{{parent}}/locations/{{location}}/buckets/{{bucket}}/links/{{link_id}}'
+import_format: ['{{%parent}}/locations/{{location}}/buckets/{{bucket}}/links/{{link_id}}']
+description: |
+  Describes a BigQuery linked dataset
+async: !ruby/object:Api::OpAsync
+  operation: !ruby/object:Api::OpAsync::Operation
+    path: "name"
+    base_url: "{{op_id}}"
+    wait_ms: 1000
+  result: !ruby/object:Api::OpAsync::Result
+    path: "response"
+    resource_inside_response: true
+  status: !ruby/object:Api::OpAsync::Status
+    path: "done"
+    complete: true
+    allowed:
+      - true
+      - false
+  error: !ruby/object:Api::OpAsync::Error
+    path: "error"
+    message: "message"
+autogen_async: true
+custom_code: !ruby/object:Provider::Terraform::CustomCode
+  encoder: templates/terraform/encoders/logging_linked_dataset.go.erb
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'linkId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |-
+      The id of the linked dataset.
+  - !ruby/object:Api::Type::String
+    name: parent
+    description: The parent of the resource.
+    url_param_only: true
+    immutable: true
+    default_from_api: true
+    diff_suppress_func: 'compareSelfLinkOrResourceName'
+  - !ruby/object:Api::Type::String
+    name: location
+    description: 'The location of the resource. The supported locations are: global, us-central1, us-east1, us-west1, asia-east1, europe-west1.'
```

Comment:
omit supported location or link to correlated documentation (markdown is supported). These values are likely to get outdated.